interfaces/functions.yaml

```diff
@@ -2512,6 +2512,9 @@ igraph_to_prufer:
 igraph_tree_from_parent_vector:
     PARAMS: OUT GRAPH graph, INDEX_VECTOR parents, TREE_MODE type=OUT
 
+igraph_is_complete:
+    PARAMS: GRAPH graph, PRIMARY OUT BOOLEAN res
```

Comment:
@ntamas What does `PRIMARY` do?

Reply:
I guess it's not needed here, but most likely it doesn't hurt. According to the Stimulus docs (see `docs/index.mdx` in the Stimulus repo):

> The `PRIMARY` keyword can be used to designate one or more _output_ arguments of a function as "primary" outputs. Code generators may use this keyword to generate two versions of a function: one that returns only the primary outputs and one that returns all of them. Alternatively, code generators may also generate a _single_ function with an additional `details` argument letting the user decide whether the result should include only the primary results or all of them. Again, it is up to the code generator to support the `PRIMARY` keyword.

Reply:
Curently the R code generator supports this. If there are both primary and non-primary out-args, the R code generator adds a `details=FALSE` argument to the function and only the primary out-args will be returned if `details=FALSE`.

Reply:
Ah, so like `is_tree()`. Then we indeed do not need this here.

Reply:
I've removed it